src/booleans.js

```diff
@@ -1,62 +1,74 @@
 function negate(a) {
-  // your code here
-};
+  return !a;
+}
 
 function both(a, b) {
-  // your code here
-};
+  return (a && b);
+}
 
 function either(a, b) {
-  // your code here
-};
+  return (a || b);
+}
 
 function none(a, b) {
-  // your code here
-};
+  return !(a || b);
+}
 
 function one(a, b) {
-  // your code here
-};
+  if (a && b) {
+    return false;
+  }
+  if (a || b) {
+    return true;
+  }
+  if (!(a && b)) {
+    return false;
+  }
+}
 
 function truthiness(a) {
-  // your code here
-};
+  return !!a;
+}
 
 function isEqual(a, b) {
-  // your code here
-};
+  return a === b;
+}
 
 function isGreaterThan(a, b) {
-  // your code here
-};
+  return a > b;
+}
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
-};
+  return a <= b;
+}
 
 function isOdd(a) {
-  // your code here
-};
+  return (a % 2 == 1)
```

Comment:
don't need parenthesis and should use strict equality, i.e  refactor to `return a % 2 === 1;`